src/providers/__init__.py

```diff
@@ -97,6 +97,20 @@ def get_contacts(self):
         ]
         return user, friends
 
+    def get_comments(self, thread_id):
+        messages = self._quip.get_messages(thread_id)
+        comments = [
+            Message(
+                comment.get("text"), comment.get("author_id"), comment.get("author_name"),
+                comment.get("created_usec"), comment.get("updated_usec"),
+                comment.get('annotation').get('highlight_section_ids')
```

Comment:
Please use one type of quotes. Does it work message do not have highlight_section_ids section?

Reply:
OK. We need a convention, what type of quotes we use in all the code
I would suggest to use single ones `'`

And yep, it will work with empty `highlight_section_ids`, because `.get()` method will return `None` if it will not find this key in the json
And I added the default value as empty list for sections here, in the message entity. You can see that in [this line](https://github.com/SPBSTU-OrangeTeam/QuipEditor/pull/9/files/fbda69ccf1103385e7b91c5f6b4972d14d802a34#diff-c912df890ef31ceec8993edb19a5b6247444720b55b5cb1e5138189627a33b4dL4)